src/js/_store/modules/categories.js

```diff
@@ -0,0 +1,32 @@
+import axios from '@Helpers/axiosDefault';
+
+export default {
+  namespaced: true,
+  state: {
+    category: {
+      id: null,
+      name: '',
+    },
+    categoryList: [],
+  },
+  mutations: {
+    doneGetAllCategories(state, { categories }) {
+      state.categoryList = categories;
+    },
+  },
+  actions: {
+    getAllCategories({ commit, rootGetters }) {
+      axios(rootGetters['auth/token'])({
+        method: 'GET',
+        url: '/category',
+      }).then(response => {
+        if (response.data.code === 0) throw new Error(response.data.message);
+        const categories = response.data.categories.map(data => ({
+          id: data.id,
+          name: data.name,
+        }));
+        commit('doneGetAllCategories', { categories });
```

Comment:
[imo]

渡したいものが1つなのであれば、オブジェクトにする必要はないと思うので、以下のように修正して良いと思います。
また、mutations の方も修正が必要になるかと思います。

```suggestion
        commit('doneGetAllCategories', categories);
```

Reply:
なるほどです！！ありがとうございます!!削除完了しております！！